addons/website_profile/static/src/js/website_profile.js

```diff
@@ -57,23 +58,23 @@ publicWidget.registry.websiteProfileEditor = publicWidget.Widget.extend({
             return def;
         }
 
-        const $textarea = this.$("textarea.o_wysiwyg_loader");
+        const textareaEl = document.querySelector("textarea.o_wysiwyg_loader");
```

Comment:
`this.el` should also works (same remark as [here](https://github.com/odoo/odoo/pull/165087#discussion_r1705256032)) ... If it's true change it please

Reply:
Yes, You are right!! :slightly_smiling_face: